demo_notebook/openspoor_example.ipynb

```diff
@@ -12,7 +12,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
```

Comment:
Voor notebooks is het handig om de metadata/rundata te verwijderen. Kan dacht ik met een nbmake commando, een beetje a la hieronder.
pytest --nbmake path/to/your/notebooks

Reply:
Fixed